src/app-nav-bar/styled-components.js

```diff
@@ -0,0 +1,168 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import {styled, withStyle} from '../styles/index.js';
+import {StyledListItem} from '../menu/index.js';
+
+const StyledClearButton = styled<{$isFocusVisible: boolean}>(
+  'button',
+  ({$theme, $isFocusVisible}) => ({
+    boxSizing: 'border-box',
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: 'transparent',
+    color: $theme.colors.contentPrimary,
+    borderLeftWidth: 0,
+    borderTopWidth: 0,
+    borderRightWidth: 0,
+    borderBottomWidth: 0,
+    paddingTop: '0',
+    paddingBottom: '0',
+    paddingLeft: '0',
+    paddingRight: '0',
+    marginLeft: 0,
+    marginTop: 0,
+    marginRight: 0,
+    marginBottom: 0,
+    outline: $isFocusVisible ? `3px solid ${$theme.colors.accent}` : 'none',
+    outlineOffset: '-3px',
+    WebkitAppearance: 'none',
+    cursor: 'pointer',
+  }),
+);
+
+export const StyledRoot = styled<{}>('div', props => {
+  const {$theme} = props;
+  return {
+    ...$theme.typography.font300,
+    boxSizing: 'border-box',
+    backgroundColor: $theme.colors.backgroundPrimary,
+    boxShadow: $theme.lighting.shadow600,
+    width: '100%',
+    position: 'fixed',
+    left: 0,
+    top: 0,
+  };
+});
+
+export const StyledSpacing = styled<{}>('div', props => {
+  const {$theme} = props;
+  return {
+    boxSizing: 'border-box',
+    height: '100%',
+    paddingTop: $theme.sizing.scale700,
+    paddingBottom: $theme.sizing.scale700,
+    display: 'flex',
+    alignItems: 'center',
+    [`@media screen and (max-width: ${$theme.breakpoints.medium - 1}px)`]: {
```

Comment:
Is there a reason to arrange media queries like this rather than using `theme.mediaQuery.medium` or `min-width`?